externs/shaka/player.js

```diff
@@ -1630,6 +1631,11 @@ shaka.extern.LiveSyncConfiguration;
  *   jump.
  *   <br>
  *   Defaults to <code>0.5</code>.
+ * @property {number} seekSlowDelay
+ *   The minimum time duration (in seconds) before we can operate a new seek
+ *   on cast devices (Only for Chromecast)
```

Comment:
I'd say it's for devices which have slow seek time. Even now, it does not apply to Android based Chromecasts

Reply:
I updated the text let me know if you agree ? @tykus160